app/bundles/PageBundle/Translations/en_US/messages.ini

```diff
@@ -158,6 +158,7 @@ mautic.widget.unique.vs.returning.leads="Unique vs returning visitors"
 mautic.widget.dwell.times="Dwell times"
 mautic.widget.popular.pages="Popular landing pages"
 mautic.widget.created.pages="Created Landing pages"
+mautic.widget.best.tracking.pages="Best tracking pages"
```

Comment:
How does this differ from 'Popular landing page'? Best tracking doesn't really sound great grammar-wise, I'd have thought that 'most popular pages' would have been better but we seem to have that in use already two lines above?

Reply:
These stats come from tracked pages with tracking codes. I suggest using combination tracked pages to determine differences to landing pages. Other suggestions are ok to me.

Reply:
So you mean that popular.pages is only considering landing pages and this is considering all pages including those outside of Mautic?

Reply:
Just outside of Mautic. Where you put mtc.js tracking script (tracking pages).

Reply:
Hm, a way to make it understandable: 

`Popular tracked pages` or `Tracked external pages`